apps/pie-docs/src/assets/styles/_dependencies.scss

```diff
@@ -13,7 +13,7 @@
 @use 'base/layout';
 @use 'base/typography';
 
-@use '@justeat/f-cookie-banner/dist/static/en-GB.css';
+@use '../../../node_modules/@justeat/f-cookie-banner/dist/static/en-GB.css';
```

Comment:
lets looks into this together and see if with can hook into loadpaths to improve this